vip-helpers/vip-elasticsearch.php

```diff
@@ -62,18 +62,36 @@ function es_api_search_index( $args ) {
 
     $args['blog_id'] = absint( $args['blog_id'] );
 
-    $service_url = 'https://public-api.wordpress.com/rest/v1/sites/' . $args['blog_id'] . '/search';
+    $endpoint = sprintf( '/sites/%s/search', $args['blog_id'] );
+    $service_url = 'https://public-api.wordpress.com/rest/v1' . $endpoint;
+
+    $do_authenticated_request = false;
+    if ( class_exists( 'Jetpack_Client' )
+            && isset( $args['authenticated_request'] )
```

Comment:
Oh, never mind. es-wp-query has the `es_posts_request` filter which we can use (https://github.com/alleyinteractive/es-wp-query/blob/master/class-es-wp-query-wrapper.php#L1033)